test/specs.js

```diff
@@ -1,15 +1,30 @@
 /* eslint-env mocha */
 
+var BuiltinModule = require('module')
+
+// Guard against poorly mocked module constructors
+var Module = module.constructor.length > 1
```

Comment:
This `var Module` can be removed because `var Module = require('module')` can be used directly in unit tests. This ternary check was added for non-Node environments which we don't use in unit tests